packages/f-footer/src/components/IconList.vue

```diff
@@ -26,7 +26,8 @@
                         "category": "engagement",
                         "action": "footer",
                         "label": "${icon.gtm}"
-                    }`'>
+                    }`'
+                    data-test-id="footerIcon">
```

Comment:
You should have access to the [`name`](https://github.com/justeat/fozzie-components/blob/master/packages/f-footer/src/tenants/en-GB.js#L216) key of the socialicons/appstore icons array here, I'd prefer if this was used here similar to:
```suggestion
                    :data-test-id="`footerIcon-${icon.name}`">
```
So that when we come to refer to the icons in `test-utils/component-objects/f-footer.component.js`, we're not reliant on any particular order of the icons.

Reply:
Ahhh yes, great shout! I'll pick this up with Lizzie next week after the hackathon. For now though, I think we're good to merge :)